pkg/helm/handlers/handlers.go

```diff
@@ -16,10 +18,29 @@ var (
 	plog = capnslog.NewPackageLogger("github.com/openshift/console", "helm")
 )
 
+func NewHelmHandler(apiUrl string, transport http.RoundTripper) *HelmHandlers {
```

Comment:
the package name is `handler`, thus no need to repeat that in this constructor, I would say `NewContext`, because this structure represent now a context.